sintr_worker/lib/basic_info.dart

```diff
@@ -87,13 +86,31 @@ String logKey, Stream<String> stream) async {
         return result;
       }
     }
-    if (line.startsWith(data_prefix)) {
-      // Extract the info and add a result of the format
-      // <millisSinceEpoch>:Ver:<uuid>:<clientId>:<clientVersion>:<serverVersion>:<sdkVersion>
-      result[basic_info_key] = [line.substring(data_prefix.length)];
+    if (line.startsWith('Data :: ~')) {
+      int index = line.indexOf(':Ver:');
+      if (index != -1) {
+        int milliSinceEpoch = int.parse(line.substring(9, index));
+        var prefix = key_prefix;
+        var key = buildKey(prefix, milliSinceEpoch);
+        // Extract the info and add a key/value of the format
+        // basic_info_<YYYY>-<MM>-<DD>
+        // <uuid>:<clientId>:<clientVersion>:<serverVersion>:<sdkVersion>
+        result[key] = [line.substring(index + 5)];
+      }
       return result;
     }
   }
-  result[basic_info_key] = [unknown_value];
+  result[key_prefix] = [unknown_value];
   return result;
 }
+
+/// Return a key of the form <prefix><YYYY>-<MM>-<DD>
+String buildKey(String prefix, int milliSinceEpoch) {
+  var time = new DateTime.fromMillisecondsSinceEpoch(milliSinceEpoch);
+  var year = time.year.toString();
+  var month = time.month.toString();
+  var day = time.day.toString();
+  if (month.length < 2) month = '0$month';
+  if (day.length < 2) day = '0$day';
+  return '$prefix$year-$month-$day';
+}
```

Comment:
Any chance of a terminal line break? I use cat a lot during development, it does really help